src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -61,6 +63,15 @@ public function getConfigTreeBuilder()
                 ->scalarNode('name_converter')->defaultNull()->info('Specify a name converter to use.')->end()
                 ->booleanNode('enable_fos_user')->defaultValue(false)->info('Enable the FOSUserBundle integration.')->end()
                 ->booleanNode('enable_nelmio_api_doc')->defaultTrue()->info('Enable the Nelmio Api doc integration.')->end()
+                ->scalarNode('db_driver')
```

Comment:
Or just register the service if the MongoODMBundle is registered. If a user want to disable it, it can still use a compiler pass.